src/utils/blogs.ts

```diff
@@ -155,6 +158,13 @@ export const getCategoriesFromBlogs = (blogs: Pick<Blog, 'category'>[]) =>
     .map(v => v.toLowerCase())
     .sort()
 
+async function getBlogExcerpt(content: Blog['content']): Promise<Blog['excerpt']> {
+  const contentHTML = await markdownToHtml(content)
+  const parser = new DOMParser()
+  const doc = parser.parseFromString(`<html><body>${contentHTML}</body></html>`, 'text/html')
+  return doc.documentElement.textContent?.substring(0, 200)
```

Comment:
Maybe an ellipsis can be appended to make truncation more naturally
![image](https://github.com/Magickbase/nervos-official-website/assets/7271329/440301be-563b-4654-87c1-796e3efa4432)

Reply:
Oh, i did not consider the case of insufficient length. However, directly adding ellipses may sometimes result in double ellipses (automatically appended by the software).
We can consider increasing the value of the excerpt to avoid this issue. The original value of 200 was designed for the website's own list pages, so now we can consider changing it to 300 or 400.
Alternatively, we can use a more appropriate paragraph truncation for the `og:description` instead of character count truncation (I observed this on https://archive.nervos.org/).

Reply:
> Oh, i did not consider the case of insufficient length. However, directly adding ellipses may sometimes result in double ellipses (automatically appended by the software).

Make sense

> We can consider increasing the value of the excerpt to avoid this issue. The original value of 200 was designed for the website's own list pages, so now we can consider changing it to 300 or 400.

This issue can be avoided by abiding to this writing style. So no update on code is needed.